indico/modules/events/contributions/operations.py

```diff
@@ -62,6 +62,7 @@ def _ensure_consistency(contrib):
 
 
 def create_contribution(event, contrib_data, custom_fields_data=None, session_block=None, extend_parent=False):
+    user = session.user if session else None
```

Comment:
is it only for making tests run?

Reply:
It also lets you use this operation from the Indico shell without creating a request context.